src/buffer.rs

```diff
@@ -607,8 +606,8 @@ mod test {
         for mut p in a.pixels_mut() {
             let rgb = p.channels_mut();
             rgb[0] = 255;
-            rgb[1] = rand::random();
-            rgb[2] = rand::random();
+            rgb[1] = 23;
```

Comment:
Totally random. ;) Most important thing is, that the compiler does not seem to optimize the stuff away although it is constant.